app/controllers/api/openid_connect/authorizations_controller.rb

```diff
@@ -0,0 +1,106 @@
+module Api
+  module OpenidConnect
+    class AuthorizationsController < ApplicationController
+      rescue_from Rack::OAuth2::Server::Authorize::BadRequest do |e|
+        logger.info e.backtrace[0, 10].join("\n")
+        render json: {error: e.message || :error, status: e.status}
+      end
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.